modules/backend/src/main/scala/smithy4s_codegen/Frontend.scala

```diff
@@ -1,12 +1,10 @@
 package smithy4s_codegen
 
 import cats.effect._
-import cats.syntax.all._
```

Comment:
thought: a bunch of unused imports here and there, I can move that to a separate PR if need be

Reply:
jeez how many of those did I leave behind
thanks for cleaning up my mess

I was convinced I had sbt-tpolecat taking care of this